File: src/projectone/ProjectOne.java
package projectone;

/**
 *
 * @author alfarkhan
 */
public class ProjectOne {
    
    public static void main(String[] args) {

        Keramik A = new Keramik();
        A.kategori = "A";
        A.ukuran = 30*30;
        A.isi = 10;
        A.harga = 54000;
        A.hitungHarga();
        
        Keramik B = new Keramik();
        B.kategori = "B";
        B.ukuran = 40*40;
        B.isi = 5;
        B.harga = 65000;
        B.hitungHarga();
        
        Keramik C = new Keramik();
        C.kategori = "C";
        C.ukuran = 30*40;
        C.isi = 8;
        C.harga = 60000;
        C.hitungHarga();
    }
    
}

File: ProjectOne.py
class  Keramik():

	def __init__(self):
		global ukuran, isi, harga
		self.kategori = {"A":[30*30,10,54000],"B":[40*40,5,65000],"C":[30*40,8,60000]}

	def hitung(self,kategori):
		## 100m2 = 1000000cm2
		data = self.kategori[kategori]
		for i in range(len(data)):
			jumlah_keramik = 1000000//data[0]
			jumlah_box = jumlah_keramik//data[1]
			harga_total = jumlah_box*data[2]

		return harga_total

def main():
	pakbejo = Keramik()
	print("A : ",pakbejo.hitung("A"))
	print("B : ",pakbejo.hitung("B"))
	print("C : ",pakbejo.hitung("C"))

if __name__ == "__main__":
	main()
